fuse-products/src/main/java/software/tnb/product/application/App.java

```diff
@@ -34,6 +41,9 @@ public abstract class App {
     protected Endpoint endpoint;
     protected boolean started = false;
 
+    private static final String JBANG_SCRIPT_NAME = "camel";
+    protected boolean hasCamel = false;
```

Comment:
:thinking: `isCamelInPath` or `isCamelPresentInPath` would be more fitting, also it could be static so the check is performed only once per testsuite run, I'd hope the path won't change during the execution :D

Reply:
thanks, it was static before, but when moving it to `App` I accidentally removed it - I'd hope too that if it was once in path it will stay like that :grinning: